src/TorchSharp/FFT.cs

```diff
@@ -350,6 +350,10 @@ public static Tensor ifftshift(Tensor input, long[] dim = null)
             /// <param name="requires_grad">If autograd should record operations on the returned tensor.</param>
             public static Tensor fftfreq(long n, double d = 1.0, torch.ScalarType? dtype = null, torch.Device device = null, bool requires_grad = false)
             {
+                if (device is null)
+                {
+                    device = get_default_device();
+                }
                 device = torch.InitializeDevice(device);
```

Comment:
Should we also modify the behavior of `torch.InitializeDevice(null)`?

(And then the 
```csharp
if (device is null)
{
    device = get_default_device();
}
```
before is no need.)

Reply:
Good idea!